conftest.py

```diff
@@ -90,12 +93,15 @@ def small_cluster(request):
 
 
 @pytest.fixture
-def small_client(small_cluster):
+def small_client(small_cluster, s3, s3_report_url, request, tmp_path):
     with Client(small_cluster) as client:
         small_cluster.scale(10)
         client.wait_for_workers(10)
         client.restart()
-        yield client
+        local_file = str(tmp_path / "preformance-report.html")
+        with performance_report(local_file):
```

Comment:
If we use the `coiled` version of performance reports, we should get hosting for free

Reply:
Oof, good point, I had forgotten about that